vcpkg/consume/binary-caching-default.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Default local vcpkg binary cache"
+description: Learn about the default vcpkg binary cache
+author: vicroms
+ms.author: viromer
+ms.date: 9/25/2023
+ms.prod: vcpkg
+ms.topic: concept-article
```

Comment:
I thought we were putting all the concept articles in the concepts section in the ToC? This one is placed in the consume section.

Reply:
I decided to put it here so it can be found with the other tutorials, this article is kind of in-between a concept and a tutorial.

It explains what the default binary cache is (which makes it conceptual), but it also teaches how to redirect or disable the default binary cache. Since restoring and disabling only takes one step each, they don't feel like actual tutorials.